src/frontend/src/components/ChangeRequestRow.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { Typography, Grid, useTheme } from '@mui/material';
+import { Box } from '@mui/system';
+import { ChangeRequest } from 'shared/src/types/change-request-types';
+import ChangeRequestDetailCard from './ChangeRequestDetailCard';
+
+interface ChangeRequestRowProps {
+  title: string;
+  changeRequests: ChangeRequest[];
+  noChangeRequestsMessage: string;
+}
+
+const ChangeRequestRow: React.FC<ChangeRequestRowProps> = ({ title, changeRequests, noChangeRequestsMessage }) => {
+  const theme = useTheme();
+
+  const displayCRCards = (crList: ChangeRequest[]) => (
+    <Box
```

Comment:
tbh same for this one, I don't think these pieces are distinct enough to separate them out into helper components, I would just put them all in the return